contrib_bots/lib/CommuteBot/CommuteBot.py

```diff
@@ -0,0 +1,215 @@
+from __future__ import print_function
+import requests
+from six.moves import configparser as cp
+import datetime as dt
+from os.path import expanduser
```

Comment:
Conventionally there is an expected grouping of Python imports: standard library imports, related third party imports and local application/library specific imports:

https://www.python.org/dev/peps/pep-0008/#imports

Also having them in alphabetical order will look good too.